doc/decisions/decision_process.md

```diff
@@ -0,0 +1,108 @@
+# Decision Process
+
+## Problem
+
+Simply discussing in an issue and then doing an implementation is okay for non-substantial changes.
+But substantial decisions must be made transparently and in a participative way.
+
+## Constraints
+
+- All relevant information about decisions must be within Elektra's repository.
+- All decisions must go through at least two review rounds, with a merge in between.
+- At least two people need to be in favor of the decision in both rounds.
+- [Documentation guidelines](/doc/contrib/documentation.md) apply.
+
+## Assumptions
+
+- People want to be informed about or even participate in what Elektra looks like in the future.
+
+## Considered Alternatives
+
+- Issues like https://issues.libelektra.org/4521
```

Comment:
While I agree that issue #4521 isn't the best for readability, that is purely because it is actually multiple issues combined into one. If it were split up into seperate issues it wouldn't be a problem at all IMHO.

Reply:
The reason I made a single issue, was 1) because it was easier to do 2) it's meant as an overview of what's left for new-backend. We can of course split it into multiple issues, if there is a need for discussions. Some of the question might not need much discussion (e.g. `elektraModulesInit` error needed basically no discussion)

Reply:
321a85749